### ISSUE TITLE
Комментарии к заявке

### ISSUE THREAD

**Issue by aakuptsov** [Fedor1311/CoursePlatform#1]
1. Заявочный файл не должен быть docx-файлом, он должен быть оформлен в соответствии с требованиями, которые можно посмотреть на ВК. Без этого заявка зачёт не получит.

2. На Степике немало курсов для школьников.

3. Есть Фоксфорд и ещё много других ресурсов для подготовки по теме. 

4. >>Передача будет осуществляться через удаленный FTP-сервер

Не очень понятно, зачем это. У любых современных онлайн-образовательных систем есть админка. 

5. >>Запись курса на камеру

Возможно, это интересно. Что имеется в виду?

Вывод: проект неоригинален и требуется существенно переработать функционал проекта.

**Comment by Fedor1311** [comment]
Здравствуйте. Комментарии принял к сведению, формат заявочного файла переделаю. Но  я с вами не согласен по некоторым пунктам. На Степике я смотрел, там есть курсы для школьников, но их всего 6 штук (да, там есть курс подготовки к огэ по математике, но он неполный и там почти нет задач второй части); все-таки этот сайт направлен больше на студентов и учеников старшей школы. Вдобавок в моем приложении будет более емкое домашнее задание и соответствующие тесты. Фоксфорд, как я знаю, платный и там нет возможности для пользователей создавать собственные курсы. По поводу FTP-сервера, я более-менее знаком с этой технологией, и мне кажется вполне приемлемым ее использование для хранения и передачи данных. Но этот пункт под вопросом, я на 100 % не уверен в его реализации, может быть изучу более удобные способы для хранения и передачи данных. Я планирую создать собственный курс для подготовки к огэ по математике, записать его и залить на платформу (думаю что это в любом случае не будет осуждаться). В целом, я хотел сказать что мое приложение будет создано на базе Степика (сам принцип того что пользователь может создавать собственные курсы и заливать их на платформу), но конечно же видоизменю его теми функциями, которые были указаны в проектной заявке и вышесказанном. По мере создания приложения я может быть дополню его другими новыми  функциями.   Мне кажется, что проект вполне оригинален и имеет право на жизнь...

**Comment by aakuptsov** [comment]
1. >>Я планирую создать собственный курс для подготовки к огэ по математике, записать его и залить на платформу

Если эта цель - главная, проект можно квалифицировать как образовательный. У нас есть такая категория и я могу связать тебя с коллегами, которые модерируют образовательные проекты.

2. >>но конечно же видоизменю его теми функциями, которые были указаны в проектной заявке и вышесказанном

Я, если честно, так и не понял, чего нет в Степике с функциональной точки зрения, но что будет у тебя. Твоё приложение, разумеется, может отличаться **контентом**, но это не ИТ-отличие одного от другого, это именно содержательное отличие и поэтому я предлагаю подумать о переквалификации твоего проекта в образовательный. Собственно, вопрос стоит так: можно ли сделать курс по ОГЭ на Степике? Ответ, думаю, звучит как "да". 

3. >>и имеет право на жизнь...

Тут штука в том, что мы не решаем вопрос жизнь - не жизнь. Смысл этих обсуждений - сделать больно сейчас, чтобы потом не сделали больно вопросы на защите, когда уже будет поздно что-либо менять. 

Резюмирую:

1. Файл надо по-прежнему переделать

2. Предлагаю подумать о переходе в категорию образовательных проектов (в случае положительного решения, надо написать мне тут в комментариях).

**Comment by Fedor1311** [comment]
Думаю, что все-таки придется перейти на образовательные проекты. Можете пожалуйста дать контакты ваших коллег, занимающихся этим. Я хотел бы обсудить с ними возможность создания образовательного курса на степике. Я написал в систему поддержки степика по поводу того, есть ли у них какие-либо возрастные ограничения для авторов курса, но они еще не ответили.

**Comment by aakuptsov** [comment]
Я делал свой курс на Степике и не помню, чтобы там спрашивали про возраст. Кроме того, кажется, в прошлом году кто-то из 11-ых классов тоже защищал образовательную ИВР на Степике. В общем, процентов на 99, думаю, всё получится.

Ок, напишу коллегам.

Пожалуйста, переделай заявку в md-файл прямо сейчас, чтобы я смог коллегам отправить ссылку.

**Comment by Fedor1311** [comment]
Вроде переделал заявку. Только она не подходит под образовательные проекты, и кое-что я бы изменил.

**Comment by Fedor1311** [comment]
Или мне сейчас надо создать новую заявку под образовательные проекты?

**Comment by aakuptsov** [comment]
Нет, я предлагаю как можно подробнее описать свою заявку и написать Наталье Алексеевне Шабат: https://www.hse.ru/org/persons/47634680  Как вариант, можно к ней ещё подойти завтра или в другой день.

Держи потом в курсе через комментарии тут, уйдёшь ли ты на образование или останешься в ИТ, пожалуйста.
